src/Cpdf.php

```diff
@@ -3512,7 +3512,7 @@ private function readPngChunks(&$data)
                             for ($ypx = 0; $ypx < $default['info']['height']; ++$ypx) {
                                 $colorBits = imagecolorat($img, $xpx, $ypx);
                                 $color = imagecolorsforindex($img, $colorBits);
-                                $color['alpha'] = (((127 - $color['alpha']) / 127) * 255);
+                                $color['alpha'] = intval((((127 - $color['alpha']) / 127) * 255));
```

Comment:
where possible, it's better to use casting to int rather than the `intval` function
```suggestion
                                $color['alpha'] = (int) (((127 - $color['alpha']) / 127) * 255);
```